docs/source/models.rst

```diff
@@ -1600,6 +1620,37 @@ For more info, see:
     mlflow models predict --help
     mlflow models build-docker --help
 
+Environment Management Tools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MLflow currently supports the following environment management tools to restore model environments:
+
+local
+    Use the local environment. No extra tools are required.
+conda
+    Create environments using conda. conda is required.
+
+    - `conda installation instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>`_
+virtualenv
+    .. note::
```

Comment:
nit: do we need spaces around note blocks?

Reply:
I think we do :)

Reply:
Removed the space above `.. note::` as it caused the following error:

```
/home/haru/Desktop/repositories/mlflow/docs/source/models.rst:1634: WARNING: Definition list ends without a blank line; unexpected unindent.
```

Reply:
@harupy I don't see the space.